test/helpers/trigger-event.ts

```diff
@@ -1,8 +1,9 @@
-window.triggerEvent = function triggerEvent (target, event, process) {
+global.triggerEvent = function triggerEvent (target, event, process) {
```

Comment:
let's keep `window`, we should avoid changing javascript code, "improvements" should be handled at an advanced phase.